x-pack/plugins/monitoring/server/routes/api/v1/cluster/clusters.ts

```diff
@@ -6,32 +6,30 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import { LegacyRequest, LegacyServer } from '../../../../types';
 import { getClustersFromRequest } from '../../../../lib/cluster/get_clusters_from_request';
+import { getIndexPatterns } from '../../../../lib/cluster/get_index_patterns';
 import { verifyMonitoringAuth } from '../../../../lib/elasticsearch/verify_monitoring_auth';
 import { handleError } from '../../../../lib/errors';
-import { getIndexPatterns } from '../../../../lib/cluster/get_index_patterns';
+import { LegacyRequest, MonitoringCore } from '../../../../types';
 
-export function clustersRoute(server: LegacyServer) {
+export function clustersRoute(server: MonitoringCore) {
   /*
    * Monitoring Home
    * Route Init (for checking license and compatibility for multi-cluster monitoring
    */
 
   // TODO switch from the LegacyServer route() method to the "new platform" route methods
   server.route({
-    method: 'POST',
+    method: 'post',
     path: '/api/monitoring/v1/clusters',
-    config: {
-      validate: {
-        body: schema.object({
-          timeRange: schema.object({
-            min: schema.string(),
-            max: schema.string(),
-          }),
-          codePaths: schema.arrayOf(schema.string()),
+    validate: {
```

Comment:
do we need to use here the `createValidationFunction` to validate the `body` and on `cluster.ts` the `params` too?

Reply:
This file was already converted a while ago. But you're right, it doesn't yet use the same validation approach as the others. I'll use this opportunity to bring it in line. :+1: